api/tests/unit/integrations/datadog/test_unit_datadog.py

```diff
@@ -19,28 +24,46 @@ def test_datadog_initialized_correctly(base_url, expected_events_url):
     api_key = "123key"
 
     # When initialized
-    data_dog = DataDogWrapper(base_url=base_url, api_key=api_key)
+    data_dog = DataDogWrapper(
+        base_url=base_url, api_key=api_key, use_custom_source=True
+    )
```

Comment:
Well, the issue here is backwards compatibility. If existing customers that are using the datadog integration have filtering rules set up based on the previous (default) source type, then releasing this for all users would break those rules. I've added the default as False, then changed it at the model level to True so that all new instances get the custom source type name.